ci/scripts/testflight

```diff
@@ -91,87 +39,33 @@ YAML
 bosh create-release
 bosh upload-release --rebase
 
-header "Deploying to ${BOSH_ENVIRONMENT}..."
+echo "----- Deploying to ${BOSH_ENVIRONMENT}..."
 release_name=$(bosh int config/final.yml --path /final_name)
 
-header "Choosing cloud-config options"
+echo "----- Choosing cloud-config options"
 vm_type=$(bosh int <(bosh cloud-config) --path /vm_types/0/name)
 disk_type=$(bosh int <(bosh cloud-config) --path /disk_types/0/name)
 network=$(bosh int <(bosh cloud-config) --path /networks/0/name)
 
 release_final_version=$(spruce json dev_releases/*/index.yml | jq -r ".builds[].version" | sed -e "s%+.*%%")
-release_dev_version="${release_final_version}.latest"
+export RELEASE_VERSION="${release_final_version}.latest"
+echo "----- Created ${RELEASE_VERSION}"
 
-cat > tmp/deployment.yml <<YAML
----
-- type: replace
-  path: /name
-  value: ${BOSH_DEPLOYMENT}
-
-- type: replace
-  path: /releases/name=${release_name}
-  value:
-    name: ${release_name}
-    version: ${release_dev_version}
-YAML
+echo "----- Uploading stemcell"
+bosh -n upload-stemcell ../stemcell/*.tgz
```

Comment:
This relative path looks dangerous. Is there a way we can express this as an absolute path?

Reply:
fixed